packages/hash/frontend/src/blocks/page/PageTitle/PageTitle.tsx

```diff
@@ -24,14 +25,16 @@ const StyledTextarea = styled(TextareaAutosize)(({ theme }) =>
     fontSize: PAGE_TITLE_FONT_SIZE,
     lineHeight: PAGE_TITLE_LINE_HEIGHT,
     fontWeight: 500,
+    color: theme.palette.black,
 
     "&::placeholder": {
       color: theme.palette.gray[40],
       opacity: 1,
     },
 
     ":disabled": {
```

Comment:
We could revert this if we go with `readOnly` insetead of `disabled`.

Reply:
putting `color` here prevented `.disabled` to have a different color.

Reply:
I meant the disabled `:disabled` styles, not the color. Sorry the comment placement wasn't very clear